fido/client.py

```diff
@@ -0,0 +1,21 @@
+def _twisted_web_client():
+    from twisted.web import client
+    return client
+
+
+class HTTP11ClientProtocolOverride(_twisted_web_client().HTTP11ClientProtocol):
```

Comment:
I agree with bplotnick++, we should make sure not to break this condition as there is a lot depending on it. I really don't like that this file will be unsafe to import even if we put a comment there, is it possible to find a solution where we don't import web.client at all?